packages/snap-preact-demo/tests/cypress/integration/autocomplete/autocomplete.spec.js

```diff
@@ -191,12 +195,13 @@ describe('Autocomplete', () => {
 		});
 
 		it('can clear input', function () {
-			cy.get(config.selectors.website.input)
-				.first()
-				.should('exist')
-				.should('have.value', config.startingQuery || config.query)
-				.clear({ force: true })
-				.should('have.value', '');
+			cy.document().then((doc) => {
```

Comment:
I find this refactor interesting. I wonder if stability is being gained simply by using the `cy.document` promise which adds a slight delay... 🤔